.env.example

```
@@ -97,6 +97,11 @@ ENABLE_WALLET_CONTEXT_SYNC=false
 # Starknet Earn (Vesu / Starkzap): wallet Earn CTA, deposit/withdraw, activity tab
 NEXT_PUBLIC_EARN_ENABLED=false
 
+# EVM → Starknet Earn (LayerSwap bridge + Vesu). Requires NEXT_PUBLIC_EARN_ENABLED=true
+NEXT_PUBLIC_EVM_EARN_ENABLED=false
+LAYERSWAP_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LAYERSWAP_API_KEY key should go before the NEXT_PUBLIC_EVM_EARN_ENABLED key

(UnorderedKey)